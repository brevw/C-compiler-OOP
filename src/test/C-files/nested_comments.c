#include "minic-stdlib.h"

/* nested comment //  */
/* nested comment /* */


//
