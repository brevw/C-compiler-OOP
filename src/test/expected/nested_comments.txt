INCLUDE
STRING_LITERAL(minic-stdlib.h)
Lexing: pass
